<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавить категорию</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Название категории:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryNameInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>175</y>
     <width>280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addCategoryButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCategoryButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>440</width>
     <height>160</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Название</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
